.github/workflows/update_copyright.yml

```diff
@@ -0,0 +1,68 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
```

Comment:
Incorrect, but appears in some other cylc-doc files too (probably my fault), can strip later.